lib/mpl_toolkits/mplot3d/tests/test_axes3d.py

```diff
@@ -39,7 +39,19 @@ def test_invisible_axes(fig_test, fig_ref):
     ax.set_visible(False)
 
 
-@mpl3d_image_comparison(['aspects.png'], remove_text=False)
+@mpl3d_image_comparison(['invisible_grid_ticks_axis.png'], style='mpl20')
+def test_invisible_grid_ticks_axis():
+    fig = plt.figure()
+    ax = fig.add_subplot(projection='3d')
+    ax.grid(False)
+    ax.set_xticks([])
+    ax.set_yticks([])
+    ax.set_zticks([])
```

Comment:
If I'm not mistaken, this might be a bit too much, as if there are no tick locations, then there's nowhere to draw the grid, so it doesn't quite check that turning off _the grid itself_ worked.

And if that's true, we may not need a full image comparison here, but instead could do a `check_figures_equal` test between setting no ticks vs turning off the grid. Though you might have to fiddle with the tick markers themselves then, so it may not be viable.

Reply:
That's a great catch, I just tried that out and you're right! Split the new test into two, should actually be testing what we want it to now.